README.md

```diff
@@ -107,6 +107,10 @@ For other plugins, checkout the [docs/plugins](./docs/plugins/) folder for detai
 
 ## Run
 
+_Please make sure the following tips before running_
```

Comment:
I suggest we remove this part for two reasons:
- `dtm init` will tell the users exactly this
- we might need to add more info on this topic in other docs